requirements/jax/env_jax.cpu.yml

```diff
@@ -1,8 +1,10 @@
 dependencies:
   - pip:
     - -f https://storage.googleapis.com/jax-releases/jax_releases.html
-    - jax[cpu]<0.3.25
+    - jax==0.3.24
```

Comment:
Don't pin to old jax; instead fix so we can bump to later jax

Reply:
fixed, updated jax to the latest version